### PR TITLE
fix upgrade 2021.1

### PR DESCRIPTION
**Pull request by amoskong**
Currently upgrade test from 2021.1 to 2021.1.latest failed in fill_db_data check.
    
- AssertionError: query 'SELECT * FROM null_support_test_old_version
                       WHERE k = null' is not valid

We also find a recover error of scylla config:

Stderr:
cp: cannot stat '/etc/systemd/system/var-lib-scylla.mount.autobackup': No such file or directory
cp: cannot stat '/etc/systemd/system/var-lib-systemd-coredump.mount.autobackup': No such file or directory

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by amoskong** [comment]
Test the PR by https://jenkins.scylladb.com/job/enterprise-2021.1/job/Reproducer/job/rolling-upgrade-centos8-test/2/console

**Comment by roydahan** [comment]
Why does pre-commit fails?

**Review by roydahan — DISMISSED** [review]

**Comment by roydahan** [comment]
@bentsi let’s merge and backport so we can release 2021.1.3

**Comment by amoskong** [comment]
> Why does pre-commit fails?

It failed for pylint check, I already fixed the pylint issues related this PR, so it's fine to merge it.

File: upgrade_test.py [hunk 1]
```diff
@@ -66,6 +66,35 @@ def check_reload_systemd_config(node):
             raise Exception("Systemd config is changed, but not reload automatically")
 
 
+def backup_conf(node):
+    if node.is_rhel_like():
+        node.remoter.run(
+            r'for conf in $( rpm -qc $(rpm -qa | grep scylla) | grep -v contains ) '
+            r'/etc/systemd/system/{var-lib-scylla,var-lib-systemd-coredump}.mount; '
```
Comment (syuu1228) [inline upgrade_test.py:73]: it seems like run `sudo cp -v $conf $conf.autobackup` unconditionally on .mount files, but are these files generated in the test? (normally it means scylla_setup is executed).
If not, I think this cp will fail.

Comment (amoskong) [reply upgrade_test.py:73]: The command is executed in a living cluster, scylla_setup has been executed in prepare stage, so the mount configs exist.

**Comment by amoskong** [comment]
Updated the PR:  rebased to latest master.

**Review by roydahan — APPROVED** [review]

**Pull request merged**